recipes-multimedia/gstreamer/gstreamer1.0-plugins-bad_%.bbappend

```diff
@@ -0,0 +1,8 @@
+# GObject introspection needs to run some commands on the native architecture,
+# and uses qemu for this. For aarch64, these commands cause qemu to crash, so
+# we disable introspection.
+EXTRA_OECONF_aarch64 += "--disable-introspection"
+
+# Wayland support is broken, so remove this until it is fixed upstream. Fixes:
+# | make[3]: *** No rule to make target 'viewporter-protocol.c', needed by 'all'.  Stop.
+PACKAGECONFIG_remove = "wayland"
```

Comment:
Is this `PACKAGECONFIG_remove` valid for all architectures or just for aarch64?

Reply:
I'm actually not sure.. I have only seen the error on aarch64, but OTOH, I have only built on aarch64.

Reply:
@jonte I am assuming you are using the `master` PELUX branch (which is actually `morty`). I haven't seen this issue for Intel architectures, otherwise our current CI jobs would be failing, right @tobsan ?

Reply:
@GordanM I'm using master of PELUX, yeah.

Reply:
I should note that I am using gstreamer1.0 from poky, and not from meta-ivi - maybe that's a difference? I'm using gstreamer1.0-plugins-bad version 1.10.4-r0

Reply:
Yes, If this was an issue we would already have failures. For that reason, I would like this to be changed so that it only applies to aarch64 unless we KNOW it is needed for all archs.

Reply:
Alright, I'll quality the `PACKAGECONFIG_remove` w/ `_aarch64`